crates/evm/core/src/precompiles.rs

```diff
@@ -52,7 +52,7 @@ pub const PRECOMPILES: &[Address] = &[
 /// [EIP-7212](https://eips.ethereum.org/EIPS/eip-7212) secp256r1 precompile address on Odyssey.
 ///
 /// <https://github.com/ithacaxyz/odyssey/blob/482f4547631ae5c64ebea6a4b4ef93184a4abfee/crates/node/src/evm.rs#L35-L35>
-pub const ODYSSEY_P256_ADDRESS: Address = address!("0000000000000000000000000000000000000014");
+pub const ODYSSEY_P256_ADDRESS: Address = address!("0000000000000000000000000000000000000100");
```

Comment:
we deployed on 0x14 and will keep that address for odyssey, the 0x100 is the proposed RIP address, I think we should just add the precompile to both addresses instead, IIRC revm has a constant for 0x100